internal/backends/linuxkms/display/mod.rs

```diff
@@ -0,0 +1,86 @@
+// Copyright © SixtyFPS GmbH <info@slint.dev>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-1.1 OR LicenseRef-Slint-commercial
+
+use i_slint_core::api::PhysicalSize;
+
+pub trait Presenter {
+    // Present updated front-buffer to the screen
+    fn present(&self) -> Result<(), Box<dyn std::error::Error + Send + Sync>>;
+}
+
+#[cfg(any(feature = "renderer-skia-opengl", feature = "renderer-femtovg"))]
+pub mod egldisplay;
+#[cfg(feature = "renderer-skia-vulkan")]
+pub mod vulkandisplay;
+
+/// This enum describes the way the output is supposed to be rotated to simulate
+/// a screen rotation. This is implemented entirely inside the actual renderer.
+#[non_exhaustive]
+#[derive(Default, Copy, Clone, Eq, PartialEq, Debug)]
+pub enum SyntheticDisplayRotation {
+    /// No rotation
```

Comment:
I'm curious what Synthetic means there?

Reply:
I was thinking that If the display of your device has the wrong orientation, you can try to rotate it to correct that. But if you can't rotate the physical display (because it is mounted incorrectly), then you could "fake" it.

I'm not entirely happy with this name TBH.

Maybe it would be better to choose a name that's closer to what it does: `RenderingRotation`.

Reply:
`RenderingRotation` seems like a better name.

The software renderer uses an (internal for now) `WindowRotation` enum. I guess it makes sense to harmonize?

Reply:
Agreed. Will rename both.